src/core/undefined_struct.rs

```diff
@@ -301,6 +306,26 @@ impl<'a> UndefinedStructTable {
     {
         self.defined_struct_iter().collect()
     }
+
+    /// Load this structure by seeking and reading the file offsets.
+    pub fn try_load_range_from_file(
```

Comment:
The function uses the word "range" and this is probably not a good word in the function name because it is not a [RangeBounds](https://doc.rust-lang.org/std/ops/trait.RangeBounds.html).  Maybe "address" because you are loading the structure from the address?